### ISSUE TITLE
Blank lines removed from function calls

### ISSUE THREAD

**Issue by matthewwithanm** [prettier/prettier#2367]
Expected output:

```js
f(
  // hi
  adbldsaklfkljadskljasdklfklasdfkladsklfkladsfkladsklfkadlsflkadskfladsklfdsaf,
  
  // hey
  adbldsaklfkljadskljasdklfklasdfkladsklfkladsfkladsklfkadlsflkadskfladsklfdsaf,
);
```

[Actual][1]:

```js
f(
  // hi
  adbldsaklfkljadskljasdklfklasdfkladsklfkladsfkladsklfkadlsflkadskfladsklfdsaf,
  // hey
  adbldsaklfkljadskljasdklfklasdfkladsklfkladsfkladsklfkadlsflkadskfladsklfdsaf
);
```

[1]: https://prettier.io/#%7B%22content%22%3A%22f(%5Cn%20%20%2F%2F%20hi%5Cn%20%20adbldsaklfkljadskljasdklfklasdfkladsklfkladsfkladsklfkadlsflkadskfladsklfdsaf%2C%5Cn%20%20%5Cn%20%20%2F%2F%20hey%5Cn%20%20adbldsaklfkljadskljasdklfklasdfkladsklfkladsfkladsklfkadlsflkadskfladsklfdsaf%2C%5Cn)%3B%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%2C%22ast%22%3Afalse%7D%7D

**Comment by vjeux** [comment]
I can confirm that prettier is expected to keep the empty line. If someone is interested in looking into it that would be great :)

**Comment by jackyho112** [comment]
@azz, @vjeux 

Tried to look into it but didn't gain much ground. :smile:

Is this specific to accounting for empty lines within the argument list? Any pointer as to where we deal with empty lines in general right now?

I guess an empty line in the following case should not be kept?

```
f(
  // hi
  adbldsaklfkljadskljasdklfklasdfkladsklfkladsfkladsklfkadlsflkadskfladsklfdsaf,

);
```

Also, should the following example,

```
f(
  abcf,

  kmlb
);
```

be printed as 

```
f(
  abcf,
  kmlb
);
``` ?

**Comment by azz** [comment]
This was fixed in #2763, thanks @jackyho112!
